src/mediaplayer.js

```diff
@@ -1944,6 +1951,30 @@ const mediaplayer = {
         } else if (this.autoStart) {
             this.play();
         }
+
+        if(this.$container && this.config.height && this.config.height !== 'auto') {
+            this._setMaxHeight();
+        }
+    },
+
+    /**
+     * Set max height limit for container
+     * using by old media items with defined height.
+     * @private
+     */
+    _setMaxHeight: function _setMaxHeight() {
+        const $player = this.$container.find('.player');
+        const $video = this.$container.find('video.video', $player);
```

Comment:
What is the second parameter of find? Based on the documentation (https://api.jquery.com/find/) is has only one parameter.

Reply:
My bad I confused with the general lookup selector where is the second parameter is a context.